scripting/FileParsing.js

```diff
@@ -1 +1,141 @@
 //Parses through file uploaded and gets back STR Fingerprint
+  //file upload for query data
+
+
+  window.onload = () => {
+      //gets input space, adds event listener for on change, should change to on submit once html is finished
+      document.getElementById("referenceUpload").addEventListener("change", event => {
+          fileGrab(event.target.files[0]); //first file selected by user
+        });
+
+    };
+  function fileGrab(file){
+      //grabs file info as stringResults
+      let fileReader = new FileReader(); //reads content of file
+      fileReader.addEventListener("load",event => {
+      let stringResults = event.target.result;
+
+      //decide what file type it is
+      let fileType  = file.name.split(".").pop();
+
+      //initiates file parsing
+      fileParse(stringResults, fileType);
+      });
+      fileReader.readAsText(file); //reads content as text to activate load for event listener
+  }
+  function fileParse(stringResults, fileType){
+          if(fileType === "csv"){
+              //if its cvs split it into an array based upon new lines, each element of array is a row in file
+              let valuesArray = stringResults.split(/\r?\n|\r/);
```

Comment:
Aren't CSVs seperated by commas and not new lines so shouldn't you be spliting by that instead?